frontend/package.json

```diff
@@ -3,20 +3,24 @@
   "version": "0.1.0",
```

Comment:
hey haven't been making any slick front-ends for the past 4 years. When I'm pulling in new changes to the package.json, should I run `yarn` ?

Reply:
yeah just run `yarn install`